lib/utilities.h

```diff
@@ -25,8 +25,13 @@ extern "C" {
  *  @{
  */
 
-#define MB (1024 * 1024UL)
-#define GB (1024 * 1024 * 1024UL)
+#ifndef MB
+#define MB (1024UL << 10UL)
```

Comment:
@arnopo 

I have question here. Since MB and GB are so common terms, should we check values compile time of MB and GB if they are defined somewhere else ?

For example:

```
#ifndef MB
...
#else

#if (MB != (1024 * 1024))
#error "error: unexpected value of MB"
#endif

```

something like this? Since MB can be defined as 1000 * 1000. 

I am asking since want to know if it's overkill and we can simply assume that everywhere it will be defined as 1024 * 1024.

Thanks,
Tanmay